src/d3/hapticplot.d3.ts

```diff
@@ -114,22 +123,22 @@ export class Hapticplot{
     const xGrid = document.createElement('a-entity');
     xGrid.id = 'xGrid';
     this.container!.appendChild(xGrid);
-    xGrid.object3D.add(new THREE.GridHelper(50, 50, 0xffffff, 0xffffff));
-    d3.select(this.container).select('#xGrid').attr('position', '0 0 0');
+    xGrid.object3D.add(new THREE.GridHelper(1, 50, 0xffffff, 0xffffff));
+    d3.select(this.container).select('#xGrid').attr('position', '0 1 -1');
     d3.select(this.container).select('#xGrid').attr('rotation', '0 0 0');
```

Comment:
Here (and elsewhere in the file), you can cascade these (put the `.attr`s right after each other instead of reselecting)